docs/module_template.md

```diff
@@ -8,20 +8,13 @@ modules:
     # -*- coding: utf-8 -*-
     # Copyright: (c) 2021, XLAB Steampunk <steampunk@xlab.si>
     #
-    # GNU General Public License v3.0+
-    # (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+    # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
     from __future__ import absolute_import, division, print_function
 
     __metaclass__ = type
 
-    ANSIBLE_METADATA = {
-        "metadata_version": "1.1",
-        "status": ["stableinterface"],
-        "supported_by": "certified",
-    }
-
```

Comment:
Is it OK to remove this? We kept this piece of information in for the sake of Ansible 2.9 users. `ansible-doc` 2.9 will mark the module without this block as community-maintained, which is not entirely true in this case where we are planing on getting collection certified.

Reply:
Yes, This is OK to remove. With the introduction of the collection in 2.9, ANSIBLE_METADATA is no longer required. You can check this documentation - https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/dev_guide/developing_modules_documenting.rst#ansible_metadata-block. It won't harm if it is there but again all other collections already removed `ANSIBLE_METADATA`.

Reply:
It is not required, but if the metadata block is not there, ansible-doc will report module as community-maintained. Is this OK for servicenow collection that will be certified?

Reply:
This is what `ansible-doc` renders if the metadata is not present:

```
AUTHOR: Manca Bizjak (@mancabizjak), Miha Dolinar (@mdolin), Tadej Borovsak (@tadeboro)
        METADATA:
          status:
          - preview
          supported_by: community
```

With metadata in place, we get back this:

```
AUTHOR: Manca Bizjak (@mancabizjak), Miha Dolinar (@mdolin), Tadej Borovsak (@tadeboro)
        METADATA:
          status:
          - stableinterface
          supported_by: certified
```

Reply:
Do we have any new information about the feasibility of dropping the metadata block from modules?